GeoParquetMaker/app.py

```diff
@@ -0,0 +1,143 @@
+import os
+import logging
+import uuid
+import subprocess
+import threading
+import requests
+from cloudevents.http import from_http
+from flask import Flask, request
+from google.cloud import storage
+import geopandas as gpd
+
+logging.basicConfig()
+logging.root.setLevel(logging.INFO)
+
+app = Flask(__name__)
+
+def download_blob(bucket_name, source_blob_name, destination_file_name):
```

Comment:
Planning to abstract to a shared utility once everything is working

Reply:
Let's discuss this more on Tuesday. I'm realizing I'm not sure how to do this the python-y way

Reply:
For the util, I think we could build it into the BaseImage at a specific `utils` folder, and then add that to $PATH.  Let's discuss 🌈